cpp/lib/src/ZoteroHarvesterConfig.cc

```diff
@@ -62,6 +62,55 @@ std::string GetHostTranslationServerUrl() {
 }
 
 
+ZoteroMetadataParams::ZoteroMetadataParams(const IniFile::Section &config_section) {
+    static const auto PREFIX_OVERRIDE_JSON_FIELD("override_json_field_");
+    static const auto PREFIX_SUPPRESS_JSON_FIELD("suppress_json_field_");
+    static const auto PREFIX_EXCLUDE_JSON_FIELD("exclude_if_json_field_");
+
+    for (const auto &entry : config_section) {
+        if (StringUtil::StartsWith(entry.name_, PREFIX_OVERRIDE_JSON_FIELD)) {
+            const auto field_name(entry.name_.substr(__builtin_strlen(PREFIX_OVERRIDE_JSON_FIELD)));
+            fields_to_override_.insert(std::make_pair(field_name, entry.value_));
+        } else if (StringUtil::StartsWith(entry.name_, PREFIX_SUPPRESS_JSON_FIELD)) {
+            const auto field_name(entry.name_.substr(__builtin_strlen(PREFIX_SUPPRESS_JSON_FIELD)));
+            fields_to_suppress_.insert(std::make_pair(field_name,
+                                       std::unique_ptr<ThreadSafeRegexMatcher>(new ThreadSafeRegexMatcher(entry.value_))));
+        } else if (StringUtil::StartsWith(entry.name_, PREFIX_EXCLUDE_JSON_FIELD)) {
+            const auto metadata_name(entry.name_.substr(__builtin_strlen(PREFIX_EXCLUDE_JSON_FIELD)));
+            exclusion_filters_.insert(std::make_pair(metadata_name,
+                                      std::unique_ptr<ThreadSafeRegexMatcher>(new ThreadSafeRegexMatcher(entry.value_))));
+        }
+    }
+}
+
```

Comment:
Q: Do we get an error message if we misspell one of the global parameters?

Reply:
No, but therefore it would be necessary to have a register of all valid parameter names for a section and that does neither exist globally nor section-specific.

Reply:
Well i guess my last statement wasn't true... i guess that there is a list which isn't complete due to the dynamic fields which are just an allowed prefix, so this could be achieved pretty easily, but i guess we could do this in a follow-up PR.